testing/php7-rrd/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Pascal Hofmann <mail@pascalhofmann.de>
+# Contributor: Micka�l Remars <github@remars.com>
+# Maintainer: Pascal Hofmann <mail@pascalhofmann.de>
+pkgname=php7-rrd
+_pkgreal=rrd
+pkgver=2.0.1
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP$_phpver bindings to rrd tool system"
+url="https://pecl.php.net/package/rrd"
+arch="all"
+license="BSD"
```

Comment:
license is not spdx compliant

Reply:
Looks like BSD-2-Clause, with one changed word: "owner" instead of "holder". Is it still BSD-2-Clause ?

Reply:
i'd say yes